src/main/kotlin/com/tschuchort/compiletesting/KotlinCompilation.kt

```diff
@@ -475,8 +489,8 @@ class KotlinCompilation {
 
 		val k2JvmArgs = commonK2JVMArgs().also {
 			it.freeArgs = sourcePaths
-			it.pluginClasspaths = (it.pluginClasspaths?.toList() ?: emptyList<String>() + getResourcesPath())
-					.distinct().toTypedArray()
+			if (it.pluginClasspaths?.isEmpty()!!)
+				it.pluginClasspaths = arrayOf(getResourcesPath())
```

Comment:
Why are you only appending the `resourcesPath` when the `pluginClasspath` is currently empty? Shouldn't it be appended always? 
It may work in the tests now because the `resourcesPath` is the path to this jar, so it would be identical to any compiler plugins declared in this project for tests. But if this library is used as a jar dependency it may no longer work.

Reply:
Right, I gonna change it!

Reply:
Fixed!